### DISCUSSION TITLE
[灌水]

### DISCUSSION THREAD

**Discussion by 114514196** [share121/inter-knot#2540] [其他]
![Uploading Screenshot_20240817_162622.jpg…]()
今天自拍了，拍的好好看

**Comment by huiye123-Hub** [comment]
确实好看<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/b171cc5b6c8d8b6257a3bf8163fb2680_567796524796359289.png" alt="阿姬得意" width="75" >
![20240817123253](https://github.com/user-attachments/assets/c24584df-16a5-4e96-989f-a2e4d761f8d7)

**Reply by 114514196** [reply]
你那能看到图片吗为什么我看不到我自己的

**Reply by HifumiMizuhara** [reply]
我看到了
